src/content/guidelines/motion/basics.mdx

```diff
@@ -87,7 +85,7 @@ Expressive motion delivers enthusiastic and vibrant, thus more visible movement.
 
 Productivity and Expression are both essential to an interface. Reserve Expressive motion for occasional, important moments to better capture user’s attention, and offer rhythmic break to the productive experience.
 
-Our [components](../../components/overview/) have basic motion build in. Use this guidance to customize motion, combine and coordinate motion through out experiences.
+[Carbon components](../../components/overview/) have motion build in for microinteractions. However, keep in mind that the motion design of an interface still depend heavily on our users to define how the components themselves enter and exit screen, and how all the components work together. Use this guidance to customize, combine, coordinate and choreograph motion of all elements throughout experiences.
```

Comment:
@anna.gonzales @clay.crenshaw @connor.leech I’ve been kind of stuck with how to express this idea in words. So we have motion work done for button hovers, dropdown opening, etc. But we cannot design the motion of a notification’s entrance and exit. So that part of the motion design relys on our users. Is there a better way to say it?

Reply:
"Carbon components have motion built in for microinteractions. However, the motion design of the parent UI — that is, how the components interact with each other and the page itself — is up to each product's design team to implement. Use this guidance to customize, combine, coordinate, and choreograph this aspect of motion in the UI."

Reply:
Also, we don't need to capitalize "Expression" in the paragraph above this one. And maybe slightly reword the next sentence: "Reserve Expressive motion for occasional, important moments to better capture the user’s attention and offer a rhythmic break to the productive experience."

Reply:
Revision based on Shixie's feedback: "Carbon components have motion built in for microinteractions. However, the motion design of the overarching UI — that is, how the components interact with each other and enter and exit the page itself — is up to each product's design team to implement. Use this guidance to customize, combine, coordinate, and choreograph this aspect of motion in the UI.